build.gradle

```diff
@@ -1,82 +1,62 @@
 plugins {
 	id 'java'
-	id 'org.springframework.boot' version '2.1.9.RELEASE'
+	id 'org.springframework.boot' version '2.5.5'
+	id 'io.spring.dependency-management' version '1.1.0'
+	id 'idea'
+	id "org.springdoc.openapi-gradle-plugin" version "1.6.0"
+	id 'eclipse'
 }
 
 group = 'org.mifos'
 version = '0.0.1-SNAPSHOT'
-sourceCompatibility = '1.8'
+sourceCompatibility = '17'
 
 repositories {
 	mavenCentral()
-
+	maven { url "https://repo.spring.io/libs-release" }
+	maven { url "https://repository.jboss.org/nexus/content/repositories/releases" }
+	maven { url "https://jfrog.sandbox.fynarfin.io/artifactory/fyn-libs-snapshot"}
 	maven {
-		url = uri('https://fynarfin.jfrog.io/artifactory/fyn-libs-snapshot-local/')
+		url = uri('https://jfrog.sandbox.fynarfin.io/artifactory/fyn-libs-snapshot')
 	}
 }
+ext {
+	camelVersion = '3.18.1'
+	zeebClientVersion = '1.3.1'
+}
 
 dependencies {
-
-	implementation 'org.springframework.boot:spring-boot-starter-web:2.5.2'
-	implementation 'org.springframework.boot:spring-boot-starter:2.5.2'
-	implementation 'org.springframework:spring-web:5.3.19'
+	implementation 'com.google.code.gson:gson:2.8.9'
+	implementation 'org.json:json:20210307'
+	implementation 'org.mifos:ph-ee-connector-common:1.4.1-SNAPSHOT'
 	implementation 'org.apache.camel.springboot:camel-spring-boot-starter:3.4.0'
 	implementation 'org.apache.camel:camel-undertow:3.4.0'
+	implementation 'org.springframework.boot:spring-boot-starter:2.5.2'
+	implementation 'org.springframework.boot:spring-boot-starter-web:2.5.2'
+	implementation 'org.springframework:spring-web:5.3.19'
 	implementation 'org.apache.camel:camel-http:3.4.0'
-	implementation 'org.apache.camel:camel-jackson:3.4.0'
-	implementation 'org.json:json:20210307'
-	implementation 'org.mifos:ph-ee-connector-common:1.0.0-SNAPSHOT'
 	implementation 'com.amazonaws:aws-java-sdk:1.11.486'
 	implementation 'com.azure:azure-storage-blob:12.12.0'
+	implementation 'io.camunda:zeebe-client-java:1.1.0'
 	implementation 'com.fasterxml.jackson.dataformat:jackson-dataformat-csv:2.12.0'
-	implementation 'io.camunda:zeebe-client-java:8.1.1'
-	implementation 'com.google.code.gson:gson:2.8.9'
-	implementation 'commons-io:commons-io:2.11.0'
+	//implementation 'org.springframework.kafka:spring-kafka:2.5.8.RELEASE'
+	implementation 'org.apache.camel:camel-jackson:3.4.0'
+	implementation 'org.apache.camel.springboot:camel-mail-starter:3.4.0'
+	implementation('io.rest-assured:rest-assured:4.4.0')
 	compileOnly 'org.projectlombok:lombok:1.18.24'
 	annotationProcessor 'org.projectlombok:lombok:1.18.24'
-	implementation 'org.apache.httpcomponents:httpmime:4.5.13'
 
-
-	testImplementation 'io.cucumber:cucumber-java:7.8.1'
-	testImplementation 'io.cucumber:cucumber-spring:7.8.1'
 	testImplementation 'org.apache.camel:camel-test-spring-junit5:3.4.0'
 	testImplementation 'io.cucumber:cucumber-junit:7.8.1'
+	testImplementation 'io.cucumber:cucumber-spring:7.8.1'
```

Comment:
remove test implementation if you are not adding unit tetsts. also  scand and remove any other unused dependency

Reply:
Removed